src/get-slack-magic-link.js

```diff
@@ -3,14 +3,22 @@ const puppeteer = require('puppeteer');
 async function getMagicLink(workspace, email, password, debug) {
   const browser = await puppeteer.launch({
     headless: !debug,
-    slowMo: debug ? 250 : 0
+    slowMo: debug ? 50 : 0
   });
   const context = browser.defaultBrowserContext();
   context.overridePermissions(`https://${workspace}.slack.com`, ['clipboard-read']);
   const page = await browser.newPage();
 
   await page.goto(`https://${workspace}.slack.com/ssb/signin_redirect/fallback`, { waitUntil: 'networkidle2' });
 
+  try {
+    await page.waitForTimeout(5000);
+    await page.waitForSelector('#onetrust-accept-btn-handler');
```

Comment:
By default, I believe Puppeteer's WaitFor timeout is [30 seconds](https://pptr.dev/#:~:text=value.-,page,.WaitTimeoutOptions).

Could we make this time much shorter such that? In the case that we don't run into the cookie popup, the command will take at least 35 seconds to run.

Likewise, we should combine the two `waitFor` commands into a single `waitForSelector` with a maximum timeout for simplicity (i.e. no reason to hard-code a 5-second wait if the popup appears within 2 seconds).